ga/20.0.0.10/kernel/Dockerfile.ubi.adoptopenjdk11

```diff
@@ -41,6 +41,10 @@ RUN yum -y install shadow-utils unzip wget findutils openssl \
     && useradd -u 1001 -r -g 0 -s /usr/sbin/nologin default \
     && LIBERTY_URL=${LIBERTY_URL:-$(wget -q -O - https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/index.yml  | grep $LIBERTY_VERSION -A 6 | sed -n 's/\s*kernel:\s//p' | tr -d '\r' )}  \
     && wget $DOWNLOAD_OPTIONS $LIBERTY_URL -U UA-IBM-WebSphere-Liberty-Docker -O /tmp/wlp.zip \
+    && LICENSE_BASE=$(wget -q -O - https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/index.yml  | grep $LIBERTY_VERSION -A 6 | sed -n 's/\s*license:\s//p' | sed 's/\(.*\)\/.*/\1\//' | tr -d '\r') \
+    && wget ${LICENSE_BASE}en.html -U UA-IBM-WebSphere-Liberty-Docker -O /licenses/en.html \
+    && wget ${LICENSE_BASE}non_ibm_license.html -U UA-IBM-WebSphere-Liberty-Docker -O /licenses/non_ibm_license.html \
+    && wget ${LICENSE_BASE}notices.html -U UA-IBM-WebSphere-Liberty-Docker -O /licenses/notices.html \    
```

Comment:
Thanks for making this update. :+1: :heart:

A minor comment for future improvement: the [Security->Image Build section](https://github.com/docker-library/official-images/blob/c6c9d7b0b3a9a2edbcfd405a5ceed26c8d5c46ae/README.md#image-build) requires that remote artifacts be verified as much as possible, while this Dockerfile hasn't had checksums or gpg in the past, we do highly recommend that they are considered. (See the linked section for more explanation)

Reply:
thanks @yosifkit / @tianon for the tips regarding the download verification.  I've opened an issue (https://github.com/WASdev/ci.docker/issues/364) to tackle this topic.